package.json

```diff
@@ -21,12 +22,12 @@
   "dependencies": {
     "json-stringify-deterministic": "^1.0.1",
     "multihashing-async": "^2.0.1",
-    "orbit-db-identity-provider": "^0.3.1",
-    "orbit-db-io": "^1.1.2",
-    "p-do-whilst": "^1.1.0",
-    "p-each-series": "^2.1.0",
-    "p-map": "^4.0.0",
-    "p-whilst": "^2.1.0"
+    "orbit-db-identity-provider": "^0.4.1-699a573.0",
```

Comment:
A side note and not required to change for the PR, but we can use the "next" tag here (`"orbit-db-identity-provider": "next"`) which will then pull the _latest_ version that is tagged "next" in npm. This way we don't need update all the package.json files if/when we update something in the other packages. Sorry I didn't notice to mention this in the earlier PRs.